packages/wallets/src/features/accounts/screens/PoaScreen/PoaScreen.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { AddressSection } from '../AddressSection';
+import { DocumentSubmission } from '../DocumentSubmission';
+import './PoaScreen.scss';
+
+const PoaScreen: React.FC = () => (
+    <div className='wallets-poa'>
+        <AddressSection />
+        <DocumentSubmission />
```

Comment:
Did this so its easier to port it out if needed. All the logics are handled inside this component itself.